lib/flash_flow/data/base.rb

```diff
@@ -25,7 +25,7 @@ def initialize(branch_config, filename, git, opts={})
       def initialize_collection(branch_config)
         stored_collection = Collection.from_hash(stored_branches)
 
-        if ! branch_config.empty?
+        if branch_config && !branch_config.empty?
```

Comment:
```suggestion
        if !branch_config&.empty?
```

Reply:
Or `branch_config.present?` since The tb-bjb(https://rubygems.org/gems/tb-bjb) is just added in this PR, and activesupport is now available. It's probably fine to add a separate activesupport so it won't depend on `tb-bjb` in case it would be removed in future (which is unlikely). Though we only have used `&.` in three places, and it's good to use it here too :)